frontend/src/components/CombatResults.vue

```diff
@@ -39,7 +39,8 @@
         <b-row>
           <b-col class="earned">
             <h4>
-              {{formattedSkill}}  {{formattedUsd}}
+              {{formattedSkillNoIGO}} {{formattedInUsd(calculateSkillPriceInUsd(calculateSkillRewardNoIGO()).toFixed(4))}} UNCLAIMED SKILL AND
+              {{formattedSkillIGOReward}} {{formattedInUsd(calculateSkillPriceInUsd(calculateSkillIGOReward()).toFixed(4))}} IGO
```

Comment:
Is the case handled if IGO isn't returned after the event? I'd rather not have to delete this back out afterwards?